static/docs.css

```diff
@@ -108,7 +108,7 @@ a.sc-logo img {
   font-size: 16px;
 }
 
-.side-nav .nav-header a, .side-nav .nav-header span {
```

Comment:
I also played around with replacing the the full `nav-header` span with a link, but that created some artifacts during page load, where titles disappeared for a handful of milliseconds.